src/pymor/algorithms/lyapunov.py

```diff
@@ -1,5 +1,3 @@
-# This file is part of the pyMOR project (https://www.pymor.org).
-# Copyright pyMOR developers and contributors. All rights reserved.
```

Comment:
This should be reverted.

Reply:
[pymepack-0.9.1-cp38-cp38-linux_x86_64.zip](https://github.com/pymor/pymor/files/12474200/pymepack-0.9.1-cp38-cp38-linux_x86_64.zip)
Please, change the extension to `.whl` and use `pip install` to install pymepack

Reply:
[pymepack-0.9.1-cp38-cp38-manylinux_2_31_x86_64.zip](https://github.com/pymor/pymor/files/12514118/pymepack-0.9.1-cp38-cp38-manylinux_2_31_x86_64.zip)
@pmli , please, use this wheel to pip install the pymepack project.